nsqworker/nsqwriter.py

```diff
@@ -7,6 +7,8 @@
 import nsq
 from nsq import Error
 
+BYTES_MAX_SIZE = 1048576
```

Comment:
Just a thought... maybe we would like to be able to tweak this value for some reason.
It won't hurt to try to read it from an env variable and fallback to this default.